deps/rabbit/src/rabbit_db_m2k_converter.erl

```diff
@@ -14,17 +14,78 @@
 -include_lib("khepri_mnesia_migration/src/kmm_logging.hrl").
 -include_lib("rabbit_common/include/rabbit.hrl").
 
+%% Functions for `rabbit_db_*_m2k_converter' modules to call.
+-export([with_correlation_id/2,
+         get_store_id/1]).
+
+%% `mnesia_to_khepri_converter' callbacks.
 -export([init_copy_to_khepri/4,
          copy_to_khepri/3,
-         delete_from_khepri/3]).
+         delete_from_khepri/3,
+         finish_copy_to_khepri/1]).
+
+-define(MAX_ASYNC_REQUESTS, 64).
```

Comment:
@kjnilsson do you have a good intuition for how many in-flight `ra:pipeline_command/4`s we should allow? I picked this so that mnesia->khepri migration would be a little faster in my local testing than import_definitions (a worker pool of 32 workers) but I don't want to accidentally overload Ra if I'm setting this too high

Reply:
The value of `64` sounds reasonable to. We can generally assume that this migration will introduce cluster load, and the operators should plan the migration based on that. That is, migrate during a less busy period.